### DISCUSSION TITLE
Routing: Add SetCumulVarPiecewiseLinearCost() to the swig wrapper

### DISCUSSION THREAD

**Discussion by Mizux** [google/or-tools#2995] [Routing (and legacy CP) questions]
**What version of OR-Tools and what language are you using?**
Version: master/v9.0
Language: Java/Python/C#

**Which solver are you using ?**
Routing Solver

**What operating system (Linux, Windows, ...) and version?**
All

**What did you expect to see**
method is wrapped

**What did you see instead?**
methods are disabled by a preprocessor directive `#ifndef SWIG` ...

https://github.com/google/or-tools/blob/b37d9c786b69128f3505f15beca09e89bf078a89/ortools/constraint_solver/routing.h#L2498-L2512

**Todo Task**
* [ ] Be able to wrap the PiecewiseLinearFunction object
* [ ] Enable wrap for these methods

**Comment by anghy23** [comment]
Waiting for a PiecewiseLinearFunction wrap, is it possible to add a couple of overloading of SetCumulVarPiecewiseLinearCost() taking in input a fixed number of parameters representing the PiecewiseLinearFunction? Iternally it could use PiecewiseLinearFunction::CreateFullDomainFunction() to build the right object.

e.g.: SetCumulVarPiecewiseLinearCost(int64_t index, int64_t initial_level, int64_t x1,
     int64_t  slope1, int64_t x2, int64_t  slope2,  int64_t x3, int64_t  slope3,  int64_t  slope_final),
